packages/iov-bns/src/encode.ts

```diff
@@ -479,15 +476,20 @@ export function buildMsg(tx: UnsignedTransaction): codecImpl.bnsd.ITx {
 
 export function buildUnsignedTx(tx: UnsignedTransaction): codecImpl.bnsd.ITx {
   const msg = buildMsg(tx);
+  const feePayer = isMultisigTransaction(tx)
+    ? // The chainId is irrelevant because we only need the data
+      conditionToAddress("xxx" as ChainId, multisignatureCondition(tx.multisig[0]))
```

Comment:
We could pull out a `conditionToWeaveAddress` out of `conditionToAddress` that returns the 20 bytes directly. Since this is @iov/bns internal only, it does not affect caller code where addresses are always bech32 encoded.